index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
Remove the emoji's and change the ! to a . should add "please click" or other instructions on how to refresh if required
```suggestion
<h2 hidden>Game over, please click refresh to play again.</h2>
```